portal-api/src/scheduler/jobs/helpers/utilisation-report-helpers.js

```diff
@@ -22,6 +29,17 @@ const getFormattedReportDueDate = async () => {
   return format(reportDueDate, 'd MMMM yyyy');
 };
 
+/**
+ * Returns the utilisation report chaser date for the current month - i.e. the date that a follow-up email should be
+ * sent to the bank to chase a report if not received by the due date
+ * @returns {Promise<Date>}
+ */
+const getReportOverdueChaserDate = async () => {
+  const bankHolidays = await externalApi.bankHolidays.getBankHolidayDatesForRegion('england-and-wales');
```

Comment:
should 'england-and-wales' be a constant somewhere so it's easy to update if the format of the JSON ever changes? (I know this is after having done this multiple times and i've only just caught it)